src/main/java/school/lemon/changerequest/java/matrix/Matrix.java

```
@@ -0,0 +1,117 @@
+package school.lemon.changerequest.java.matrix;
+
+import java.util.Random;
+
+public class Matrix {
+    private int value[][];
+    private int row;
+    private int column;
+
+    public Matrix add(Matrix matrix) {
+        if (!isSameSize(matrix))
+            return null;
+        Matrix result = new Matrix(this);
+        for (int i = 0; i < row; i++)
+            for (int j = 0; j < column; j++)
+                result.value[i][j] += matrix.value[i][j];
+        return result;
+    }
+
+    public Matrix subtract(Matrix matrix) {
+        if (!isSameSize(matrix))
+            return null;
```

Throw an exception instead of returning `null`